app/views/twoeye_content_review.scala.html

```diff
@@ -0,0 +1,71 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import models.{PageReview, ApprovalProcessReview}
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(layout: main_layout, govukTable : GovukTable)
+
+@(review: ApprovalProcessReview)(implicit request: Request[_], messages: Messages)
+
+@layout(messages("service.name")) {
+
+    <header class="page-heading">
+      <span class="govuk-caption-xl">@messages("2iReview.heading")</span>
+      <h1 class="govuk-heading-xl">@review.title</h1>
+    </header>
+
+    
+    <ol class="app-task-list">
+      <li>
+
+        <h2 class="app-task-list__section">
+          <span class="app-task-list__section-number">1. </span>@messages("2iReview.reviewPagesHeading")
+        </h2>
+
+        <ul class="app-task-list__items">        
+          @for((page, index) <- review.pages.zipWithIndex) {
+            <li class="app-task-list__item">
+              <a href="#" class="app-task-list__task-name" aria-describedby="page-@index">@page.title</a>
+              <strong class="govuk-tag app-task-list__task-completed" id="page-@index">@messages(s"pageReviewStatus.${page.status.toString}")</strong>
```

Comment:
I would expect there to be a class for "task not completed", but I can't see one in the CSS.

Reply:
Yes, but unfortunately its not defined anywhere. If I remove this class on incomplete tasks all the formatting is wrong.

Reply:
This might be something to do with the task list not having an initial status shown on the example - it goes from blank, to Complete, so only has a style for completed.  That's another area our page design differs from the template example (as well as the link on last section)

Reply:
Yes, the design standard advises displaying no status until the task is complete